### PR TITLE
Support big data for insert-js

### PR DESCRIPTION
**Pull request by namse**
@bigfoodK please test this with psd. you don't have to chunk psd file anymore with this changes

### CODE REVIEW

**Review by bigfoodK — CHANGES_REQUESTED** [review]
Error occurs.
```text
Uncaught (in promise) Error: Unknown event! {"type":"insert-js/request-data-buffer","jsId":["u32",0],"bufferLen":["u32",4],"requestId":["u32",0]}
    at NewEventSystemHandleOnMainThread.sendEvent (newEventSystem.ts:147:23)
    at window.namui_onData (insertJs.ts:72:30)
    at namui_sendData (<anonymous>:2:24)
    at inputElement.onchange (<anonymous>:43:5)
```

Maybe changes of `namui-cli/webCode/src/newEventSystem.ts` are missing.

**Comment by namse** [comment]
Please test again

File: namui/namui-cli/webCode/src/insertJs.ts [hunk 1]
```diff
@@ -1,155 +1,137 @@
...
-            const onDropKey = `namui_onDrop_${id}`;
+            const onDropKey = `namui_onDrop_${jsId}`;
             (window as any)[onDropKey]?.();
             delete (window as any)[onDropKey];
             js.script.remove();
-            jsMap.delete(id);
+            jsMap.delete(jsId);
+        },
+
+        OnInsertJsDataBuffer({
```
Comment (bigfoodK) [inline namui/namui-cli/webCode/src/insertJs.ts:110]:
```suggestion
        onInsertJsDataBuffer({
```

File: namui/namui-cli/webCode/src/insertJs.ts [hunk 2]
```diff
@@ -1,155 +1,137 @@
...
+            const js = jsMap.get(jsId);
+            if (!js) {
+                throw new Error(`No js for ${jsId} found`);
+            }
+            const data = js.dataBufferRequestMap.get(requestId);
+            if (!data) {
+                throw new Error(`No data for ${requestId} found`);
+            }
+
+            new Uint8Array(data).set(new Uint8Array(memory.buffer, bufferPtr));
```
Comment (bigfoodK) [inline namui/namui-cli/webCode/src/insertJs.ts:126]:
```suggestion
            new Uint8Array(memory.buffer, bufferPtr).set(new Uint8Array(data));
```

Reversed

**Review by bigfoodK — CHANGES_REQUESTED** [review]

**Review by bigfoodK — APPROVED** [review]

**Pull request merged**